docs/cep/query-guide/custom-script-functions.md

```diff
@@ -0,0 +1,146 @@
+---
+sidebar_position: 80
+title: Custom Script Functions
+---
+
+Scripts allow you to write functions in JavaScript and execute them within queries for a stream. Functions defined with scripts can be accessed in queries similar to any other inbuilt function. You can use function definitions to define these scripts.
+
+Function parameters are passed into the function logic as an `Object[]` and with the name `data`.
```

Comment:
```suggestion
Function parameters are considered an `Object[]` with the name `data`.
```

Do we need to specify `Object[]` syntax instead of just saying "considered an object" or "considered objects"